ovs/lib/helpers/toolbox.py

```diff
@@ -197,3 +197,24 @@ def change_service_state(client, name, state, logger=None):
         else:
             logger.debug('  {0:<15} - Service {1} {2}'.format(client.ip, name, action))
             print '  [{0}] {1} {2}'.format(client.ip, name, action)
+
+    @staticmethod
+    def wait_for_service(client, name, status=True):
+        """
+        Wait for service to enter status
+        :param client: SSHClient to run commands
+        :param name: name of service
+        :param status: True - running/False - not running
+        """
+        ServiceManager.enable_service(name, client=client)
```

Comment:
So when somebody actually wants to wait for a service to be in a given state, we enable it. So if somebody just disabled a service, stopped is and now wants to wait for it to be actually stopped, we just enabled the service again.